openxr/examples/hello.rs

```diff
@@ -8,11 +8,11 @@ fn main() {
         .expect("couldn't find the OpenXR loader; try enabling the \"static\" feature");
 
     let extensions = entry.enumerate_extensions().unwrap();
-    println!("supported extensions: {:?}", extensions);
+    println!("supported extensions: {:#?}", extensions);
     let layers = entry.enumerate_layers().unwrap();
     println!("supported layers: {:?}", layers);
     if !extensions.khr_convert_timespec_time {
-        panic!("timespec conversion unsupported");
+        println!("warning: timespec conversion unsupported");
     }
```

Comment:
I'd just get rid of this if statement entirely; it's vestigal from when this did more using the headless extension that the prerelease spec had.

Reply:
Sure--I almost did originally but thought it might have a purpose.